tests/server/conftest.py

```diff
@@ -91,6 +94,36 @@ def auto_login_not_authorized():
     return app
 
 
+@pytest.fixture
+def ldap_app(request):
```

Comment:
I've just moved this here so it can be used by externsions/ldap_extension and server/blueprints/login/test_views